src/DynamoCore/Library/LibraryCustomization.cs

```diff
@@ -4,6 +4,7 @@
 using System.Reflection;
```

Comment:
Same decision here, `LibraryCustomizationServices` is kept as `static` for now, so `IPathManager` needs to be passed into them for now.